mne_bids/write.py

```diff
@@ -89,7 +89,8 @@ def _channels_tsv(raw, fname, overwrite=False, verbose=True):
                     ecg='ElectroCardioGram',
                     eog='ElectroOculoGram',
                     emg='ElectroMyoGram',
-                    misc='Miscellaneous')
+                    misc='Miscellaneous',
+                    bio='Biological')
```

Comment:
What **is** a bio channel?

Reply:
To be honest... I'm not quite sure. The only documentation I found was here: https://mne.tools/stable/generated/mne.create_info.html

@agramfort any idea how to handle `bio` channels?

Reply:
Any idea how to handle this besides pumping it to 'Misc'? The correct one is probably  the `Other` keyword, but then it breaks roundtrip.

Reply:
I think Misc is okay…